### ISSUE TITLE
error running stereoCell

### ISSUE THREAD

**Issue by GuangWei-Zhang** [BGIResearch/StereoCell#3]
I used the segmentation and labeling function. seems the model interference went well however could not generate the correct cell bin matrix

**Comment by niovl** [comment]
Hi, Mr zhang. 
Please tell me the size of tissue.gem.gz and registration.tif,  where registration.tif got mask.tif after doing the cell segmentation.

**Comment by GuangWei-Zhang** [comment]
Thanks for the prompt response. Do you mean the file size or the resolution of the tif file?

**Comment by GuangWei-Zhang** [comment]
Yes, I do get the regist_cell_mask.tif and the _tissue_mask.tif  files. The tissue.gem.gz is arond 240MB and the regist.tif is around 700MB.

**Comment by GuangWei-Zhang** [comment]
Demension of the regist.tif and matrix are not extractly match. te regist.tif is (26457,26459), matrix is (26458,26456). May I know how to correct this mismatch?

**Comment by GuangWei-Zhang** [comment]
@niovl

**Comment by niovl** [comment]
@GuangWei-Zhang I wonder how you got file regist.tif
